cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -362,3 +365,8 @@ ensureNewFile writer outFile blob = do
 
 ensureNewFileLBS :: FilePath -> LB.ByteString -> IO ()
 ensureNewFileLBS = ensureNewFile LB.writeFile
+
+getGenesisHash :: GenesisFile -> ExceptT CliError IO Text
+getGenesisHash genFile = do
+  (_, Genesis.GenesisHash gHash) <- readGenesis genFile
+  return $ F.sformat Crypto.hashHexF gHash
```

Comment:
is this computing the genesis' hash?
then, you can ignore my above comment about keeping the argument 'genesis-hash'

Reply:
@CodiePP  Yes this is computing the genesis hash via https://github.com/input-output-hk/cardano-ledger/blob/a773ba71eb9a6fb91dda2cdf6c41b2280f2acf7d/cardano-ledger/src/Cardano/Chain/Genesis/Data.hs#L116